lib/suspenders/app_builder.rb

```diff
@@ -366,6 +372,20 @@ def create_heroku_apps(flags)
       create_production_heroku_app(flags)
     end
 
+    def create_heroku_pipeline
+      pipeline_name = app_name.dasherize
+
+      run_heroku "pipelines:create -a #{pipeline_name}", "staging"
+      run_heroku(
+        "pipelines:add -a #{app_name.dasherize}-staging #{pipeline_name}",
+        "staging",
+      )
+      run_heroku(
+        "pipelines:add -a #{app_name.dasherize}-production #{pipeline_name}",
```

Comment:
Is the `app_name` passed in to this method?
If not, I would keep it simple for now, rather than something that will be nice in the future.
Also, if we are calling all the apps `x-staging` and `x-production` then it makes sense to call the pipeline `x`
What do you think?